tsdb/head.go

```diff
@@ -1387,19 +1381,20 @@ func (a *headAppender) Commit() (err error) {
 	}
 	defer func() { a.closed = true }()
 	if err := a.log(); err != nil {
-		//nolint: errcheck
-		a.Rollback() // Most likely the same error will happen again.
+		_ = a.Rollback() // Most likely the same error will happen again.
```

Comment:
is there a reason to change this instead of just using `//nolint: errcheck`? To me I could see `_ = func` as someone forgetting to actually check the error after prototyping phase of some code, where as `nolint` is them specifically choosing to not check the error

Reply:
This is exactly the same no? You chosen to not check the error. The difference is that you use native Go feature to say so, instead of linter that can change 🤗

Reply:
I suppose so. Seeing a `//nolint` comment to me is just clearer that the author intended to not check the error, instead of just forgot to.